test_dancho.py

```diff
@@ -27,11 +28,16 @@ def say_time():
 
 
 def say_date():
-    """"Type your code here"""
-    pass
+    cur_date = datetime.date.today().strftime("%d:%m:%Y")
+
+    spec_dates = {"17:03", }
+    spec_dates_prompts = {"17:03": f"It's my birthday."}
 
-print("Hello world!")
+    if not spec_dates.issuperset(cur_date[:-5]):
+        speak(f"Today's {cur_date}. It's just a regular day")
```

Comment:
On this line, use the `return` statement  to send the string as as result from the function `say_date` and call the function on the last line